kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -700,6 +703,19 @@ impl<'tcx> GotocCtx<'tcx> {
         self.codegen_fn_item(instance, span)
     }
 
+    pub fn codegen_fndef_stable(
+        &mut self,
+        def: FnDef,
+        args: &GenericArgs,
+        span: Option<SpanStable>,
+    ) -> Expr {
+        let instance = InstanceStable::resolve(def, args).unwrap();
```

Comment:
Handle how exactly? It is a bug not being able to resolve an instance here.

Reply:
Having a user-friendly error, not the usual `failed to unwrap ...`. But I don't know if the "nice error" APIs are accessible from here.

Reply:
Ok. I'll replace the unwrap here by an expect(). I think we should still crash, since this would be a compilation bug not an error that user should act on.